From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Thu, 6 Jul 2017 19:44:44 +0200
Subject: ramips: simplify ubnt-erx-sfp device definition

Most of the ubnt-erx definition can be reused; the package removals in
DEVICE_PACKAGES have become redundant after d17cb4a68a45 "ramips: purge
default packages on MT7621".

Signed-off-by: Matthias Schiffer <mschiffer@universe-factory.net>

diff --git a/target/linux/ramips/image/mt7621.mk b/target/linux/ramips/image/mt7621.mk
index 8218bea84410536151a216cd2cc0300369ad977a..11b5539b80f35b4e4d130a5f21014c8960bf72d4 100644
--- a/target/linux/ramips/image/mt7621.mk
+++ b/target/linux/ramips/image/mt7621.mk
@@ -145,15 +145,10 @@ endef
 TARGET_DEVICES += ubnt-erx
 
 define Device/ubnt-erx-sfp
+  $(Device/ubnt-erx)
   DTS := UBNT-ERX-SFP
-  FILESYSTEMS := squashfs
-  KERNEL_SIZE := 3145728
-  KERNEL := $(KERNEL_DTB) | uImage lzma
-  IMAGES := sysupgrade.tar
-  KERNEL_INITRAMFS := $$(KERNEL) | ubnt-erx-factory-image $(KDIR)/tmp/$$(KERNEL_INITRAMFS_PREFIX)-factory.tar
-  IMAGE/sysupgrade.tar := sysupgrade-tar | append-metadata
   DEVICE_TITLE := Ubiquiti EdgeRouter X-SFP
-  DEVICE_PACKAGES := -kmod-mt76 -kmod-rt2x00-lib -kmod-mac80211 -kmod-cfg80211 -wpad-mini -iwinfo kmod-i2c-algo-pca kmod-gpio-pca953x kmod-i2c-gpio-custom
+  DEVICE_PACKAGES += kmod-i2c-algo-pca kmod-gpio-pca953x kmod-i2c-gpio-custom
 endef
 TARGET_DEVICES += ubnt-erx-sfp
 
